wings/market/ddex_market.pyx

```diff
@@ -440,8 +456,61 @@ cdef class DDEXMarket(MarketBase):
             self.logger().debug(f"{data}")
             return data
 
+    async def _update_trade_fees(self):
+        cdef:
+            double current_timestamp = self._current_timestamp
+
+        if current_timestamp - self._last_update_trade_fees_timestamp > 60.0 * 60.0 or self._gas_fee_usd == NaN:
```

Comment:
Should save the 60 * 60 as a static variable at the top of the market class, along with API_CALL_TIMEOUT